configs/NBD_INFINITYAIO/config.h

```diff
@@ -69,6 +74,7 @@
 #define DEFAULT_CURRENT_METER_SCALE 230
 #define BEEPER_INVERTED
 #define FLASH_SPI_INSTANCE SPI1
+#define USE_OSD
```

Comment:
Need to remove. There are no OSD pins defined. Also this is build option
```suggestion

```

Reply:
@haslinghuis when you meant OSD you meant the OSD chip?

Reply:
This FC has no FLASH, BARO or OSD chips.